server/core/dcb.cc

```diff
@@ -128,6 +128,9 @@ void dcb_global_init()
     this_unit.dcb_initialized.state = DCB_STATE_ALLOC;
     this_unit.dcb_initialized.ssl_state = SSL_HANDSHAKE_UNKNOWN;
     this_unit.dcb_initialized.poll.handler = dcb_poll_handler;
+    this_unit.dcb_initialized.high_warter_has_reached = false;
+    this_unit.dcb_initialized.low_water = 65536;
```

Comment:
Use values from `config_writeq_high_water` and `config_writeq_low_water` here. This way the value is not assigned twice.